app/src/main/java/me/hufman/androidautoidrive/music/MusicController.kt

```diff
@@ -1,9 +1,11 @@
 package me.hufman.androidautoidrive.music
 
 import android.content.Context
+import android.graphics.Bitmap
```

Comment:
I suspect these were left over from a previous iteration :)

Reply:
It was, thanks for catching that. One of the downsides of having such large changesets for the PR is it is easy to glance over things like this, definitely why I need a separate set of eyes to look at it since I clearly missed quite a bit :)